protocol/communities_key_distributor.go

```diff
@@ -96,6 +96,7 @@ func (ckd *CommunitiesKeyDistributorImpl) sendKeyExchangeMessage(community *comm
 		MessageType:           protobuf.ApplicationMetadataMessage_CHAT_MESSAGE,
 		HashRatchetGroupID:    hashRatchetGroupID,
 		PubsubTopic:           community.PubsubTopic(), // TODO: confirm if it should be sent in community pubsub topic
+		//ContentTopicOverride:  community.MemberUpdateChannelID(), //TODO: Confirm if this is correct, could not figure out where LocalChatID is set in this flow
```

Comment:
I left it because i was not sure of this flow. But since this is a control message we can let it be on its own content-topic for now. 

But now that i think about it, i will remove this line.

Reply:
Yes, please remove it for now. Keys are sent as direct private messages. Additionally, in some cases, keys are redundantly sent in a [grouped message](https://github.com/status-im/status-go/blob/5d4838a752d2575c76d0d48669f94e5b47ef02db/protocol/common/message_sender.go#L652). The concern with LocalChatID is valid, as it ends up being empty in this flow. That will need further investigation once we switch to grouped messages only.

Reply:
> Yes, please remove it for now. Keys are sent as direct private messages

Oh, interesting. I got confused because community pubsubTopic is being used to send the keys which means they will not be Direct messages right. I thought Direct messages were sent on default shard 32.